public/data/ayushlohmod.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ayush Lohmod",
+  "type": "personal",
+  "bio": "I am Ayush Lohmod, currently pursuing a Batchelors of Computer Application from SGT UNIVERSITY. I am a tech enthusiast. I am always open to collaborating on projects and innovative/disruptive ideas",
+  "avatar": "https://github.com/ayushlohmod.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ayushlohmod",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ayushlohmod_",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/ayushlohmod",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Full Stack development ",
```

Comment:
There's an extra space here. Might want to remove it.
```suggestion
      "title": "Started Full Stack development",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient